### ISSUE TITLE
Browser start-up failure after upgrading Selenium to 4.26.0

### ISSUE THREAD

**Issue by mshabarov** [vaadin/testbench#1832]
After upgrading TestBench 9.3 (main branch) to use Selenium 4.26.0, the following test fails to create a WebDriver and start the Chrome:
```
Caused by: org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
```

`SeleniumHubPageObjectIT.findUsingValueAnnotation`.

See [Build and Deploy Testbench SNAPSHOT](https://bender.vaadin.com/buildConfiguration/VaadinProTools_TestBench6_BuildAndDeployTestbench6snapshot) for more details.

**Comment by mcollovati** [comment]
To be noted that the test is configured to run on Sauce Labs, so it could be an error on the server and not in Testbench itself

**Comment by mshabarov** [comment]
Something wrong with the SauceLabs endpoints.
I wonder if we can use another SauceLabs endpoint/url.
Maybe they are missing latest Selenium/WebDriver/Chrome version updates?

**Comment by mshabarov** [comment]
At first I was thinking that the root cause is open files limit, because when I created a tunnel with Sauce Connect, I got this:
```
[INFO] (server msg) The open file limit in your Operating System is too low. Sauce Labs recommends setting it to at least 64000. Please see https://docs.saucelabs.com/secure-connections/sauce-connect/system-requirements/ for more information.
```
but our agents already had a sufficient values.

Then we noticed that the tests are ran queued to SauceLabs in one time, and after a certain time, SauceLabs auto drops the queued tests, that apparently leads to these session/webdriver creation issues and flakiness.

The solution was to change `forkCount` parameter of `maven-failsafe-plugin` and this changed how this test queue is processed. See https://github.com/vaadin/testbench/pull/1833.
